wordpress-mobile/WordPress-Android
# Dashboard Personalization: Add more menu to todays stats card

Parent #18944 

This PR adds the "more menu" to the Todays Stats Card

Menu Closed | Menu Open
--|--
<img width="250" height="500" alt="Alt desc" src="https://github.com/wordpress-mobile/WordPress-Android/assets/506707/fa0a1d0c-119c-4c3a-818a-5e48dcf62250"> | <img width="250" height="500" alt="Alt desc" src="https://github.com/wordpress-mobile/WordPress-Android/assets/506707/110bfb92-9330-47c0-a435-a09ff53b1454">

**Included in this PR are the following:**
- Added "more menu" to the today stats dashboard card
- Added my site toolbar to the today stats card
- Handle the more menu click
- Handle navigation on more menu item clicks

Not included in this PR:
- Tracking events for the menu item taps
- Logic to hide the card when "hide this" is tapped.

**Merge Instructions**
- Ensure PR #19042 & #19043 have been merged
- Double check that the base branch is set to `feature/dashboard-personalization`
- Remove the do not merge label
- Merge as normal

**To test:**
- Install and login to the app
- Select a site in which you have stats access
- Navigate to the My Site tab
- Scroll to the Todays Stat card
- ✅ Verify the card does not contain a footer
- ✅ Verify the card has a vertical more menu
- Tap on the More Menu
- ✅ Verify the more menu is expanded
- Tap on the View Stats menu item
- ✅ Verify you are navigated to the Stats view


## Regression Notes
1. Potential unintended areas of impact
The todays stats does not work properly

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Updated unit tests

3. What automated tests I added (or what prevented me from doing so)
CardsTrackerTest

PR submission checklist:

UI Changes testing checklist:
